### PR TITLE
Adding diagram service generation endpoint to the website

### PR DESCRIPTION
**Pull request by ricardozanini**
Signed-off-by: Ricardo Zanini <zanini@redhat.com>

**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:
In this PR, we add a new redirect to avoid hard coding the service URL into the plugin. Users won't have to reinstall the vs-code plugin if this URL change in the future.

### CODE REVIEW

**Comment by ricardozanini** [comment]
Worked: https://deploy-preview-50--serverlessworkflow.netlify.app/services/diagrams/generate

**Review by tsurdilo — COMMENTED** [review]
There are two :) 
The https one is used by the html editor:

https://github.com/serverlessworkflow/serverlessworkflow.github.io/blob/master/js/swedtior.js#L445

**Comment by ricardozanini** [comment]
Cool! I'll unify everything tomorrow in this PR :dancers:

**Comment by ricardozanini** [comment]
@tsurdilo, I think we can rely only on the secured endpoint. Tested the online editor: https://deploy-preview-50--serverlessworkflow.netlify.app/editor.html. Working perfectly. :ok_hand:

File: _redirects [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # See:  https://docs.netlify.com/routing/redirects/
 #       https://docs.netlify.com/routing/redirects/redirect-options/
 /schemas/latest/* /schemas/0.5/:splat    200
-/schemas/snapshot/* /schemas/0.6-alpha/:splat    200
\ No newline at end of file
+/schemas/snapshot/* /schemas/0.6-alpha/:splat    200
+/services/diagrams/generate https://diagram-service-secure-mmagnani.apps.kogito-cloud.automation.rhmw.io/swdiagram 200
```
Comment (tsurdilo) [inline _redirects:5]: could this be like "/services/diagram"
only? i doubt we will have anything else except "generate".

sorry :)

Comment (ricardozanini) [reply _redirects:5]: Sure, no problem!

**Comment by ricardozanini** [comment]
IDK what's going on, my PR ain't gotta updated. I'll close and open a new one.

**Pull request closed without merging**